backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -447,6 +457,113 @@ def test_set_project_link(self):
         project.refresh_from_db()
         self.assertEqual(project.link_submission, link_2)
 
+    def test_get_number_of_times_card_moved_to_review_feedback(self):
+        recruit = UserFactory()
+        card1 = factories.AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+        )
+        card1.assignees.set([recruit])
+
+        card2 = factories.AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+        )
+        card2.assignees.set([recruit])
+
+        actor_user = UserFactory(is_superuser=True)
+        self.login(actor_user)
+
+        # card 1 first review, timestamp set to 2 day before
+        response = self.client.post(
+            path=f"{self.get_instance_url(pk=card1.id)}add_review/", 
+            data={"status": NOT_YET_COMPETENT, "comments": "nyc"}
+        )
+        self.assertEqual(response.status_code, 200)
+        card1.refresh_from_db()
+
+        card_moved_to_review_feedback_id = EventType.objects.filter(name=creators.CARD_MOVED_TO_REVIEW_FEEDBACK).first().id
+        c = LogEntry.objects.filter(
+            event_type=card_moved_to_review_feedback_id, 
+            effected_user=card1.assignees.first(),
+            object_1_id=card1.content_item.id,
+        ).last()
+        c.timestamp = c.timestamp - timedelta(days=2)
```

Comment:
I'm getting an error when I run the tests:

```
ERROR: test_get_number_of_times_card_moved_to_review_feedback (curriculum_tracking.tests.test_api_views.AgileCardViewsetTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/sam/Documents/Umuzi/Products/Tilde/backend/curriculum_tracking/tests/test_api_views.py", line 500, in test_get_number_of_times_card_moved_to_review_feedback
    c = LogEntry.objects.filter(
AttributeError: 'NoneType' object has no attribute 'timestamp'

----------------------------------------------------------------------
Ran 70 tests in 116.148s

FAILED (errors=1)
```

So it's not registering a matching log entry object, so there is no timestamp to update